modules/default.nix

```diff
@@ -226,7 +237,41 @@ in
       otaUtils # Tools for UEFI capsule updates
     ];
 
+    systemd.tmpfiles.rules = lib.optional nvidiaContainerRuntimeActive "d /var/run/cdi 0755 root root - -";
+
+    systemd.services.nvidia-cdi-generate = {
+      enable = nvidiaContainerRuntimeActive;
+      serviceConfig = {
+        Type = "oneshot";
+        RemainAfterExit = true;
+        ExecStart =
+          let
+            exe = "${pkgs.nvidia-jetpack.nvidia-ctk}/bin/nvidia-ctk";
+          in
+          toString [
+            exe
+            "--debug"
```

Comment:
Do we want --debug to be enabled during normal use?

Reply:
We don't need it, removed!